cmd/util.go

```diff
@@ -74,14 +74,13 @@ func mustGetIntFlag(cmd *cobra.Command, name string) int {
 	return flagVal
 }
```

Comment:
if most flags are to be removed, then these funcs are not used anymore and should be removed as well

Reply:
Done